README.md

```diff
@@ -1,6 +1,6 @@
-<p align="center">
+<h1 align="center">
     <img src="readme/alumet-banner.png" height="141px"></img>
-</p>
+</h1>
```

Comment:
This seems to create an unwanted separator after the image :S
Before (main branch):
![image](https://github.com/user-attachments/assets/e8ef103b-7555-42ae-b501-83f9d4b784d3)

After (this PR):
![image](https://github.com/user-attachments/assets/cd087f9d-ebe4-41d5-ae52-f622fd5b1c68)

Reply:
If you get the same in the local preview, you can keep the `h1` and remove the `#` which only existed to create the separator.